test/test-xterm256.rb

```diff
@@ -44,33 +44,43 @@ def test_ansi_color_code(code)
     assert { Colors::Xterm256.new(16) != Colors::Xterm256.new(17) }
   end
 
-  data do
-    data_set = {}
-    # colors 16-231 are a 6x6x6 color cube
-    (0...6).each do |r|
-      (0...6).each do |g|
-        (0...6).each do |b|
-          code = 6*(6*r + g) + b + 16
-          red   = (r > 0) ? 40*r + 55 : r
-          green = (g > 0) ? 40*g + 55 : g
-          blue  = (b > 0) ? 40*b + 55 : b
-          label = "Color #{code} is rgb(#{red}, #{green}, #{blue})"
-          data_set[label] = [code, Colors::RGB.new(red, green, blue)]
-        end
+  RGB2GREY = [0.21263900587151035754, 0.71516867876775592746, 0.07219231536073371500]
+
+  # TODO: fix test-unit for keeping data when the data given from a block
```

Comment:
@kou test-unit doesn't keep data when the data is given from a block.  We must fix it.

Reply:
I've fixed it in test-unit 3.5.1.

Reply:
Thanks!!